src/utils/vscode/getPRsToPaintPerSHAs.ts

```diff
@@ -34,8 +37,22 @@ export default async function getPRsToPaintPerSHAs({
   // takes the first 22 shas and creates a list to send to the gh api
   let joinedArrayOfSHAs = arrayOfSHAs.slice(0, 22).join();
   if (joinedArrayOfSHAs.length < 1) {
-    noLinesSelected();
-    return { errorText: "No lines selected" };
+    // Running an experiment to see how users behave without this error prompt
+    // But rather running a git blame for the whole class
+    // noLinesSelected();
+    // return { errorText: "No lines selected" };
+    let gitAPI = await getGitAPI();
+
+    const linecount = vscode.window.activeTextEditor?.document.lineCount as number;
+
+    arrayOfSHAs = await getSHAArray(
```

Comment:
This is one of the params of this function, why set it again?

Reply:
It probably will change the parent's value

Reply:
Are you talking about `arrayOfSHAs`? If so, I agree we might need another name just in case something weird with the control flow happens. I mean, the problem you mentioned shouldn't happen because this is encapsulated inside an if, but I agree.

That being said, we do need to modify `joinedArrayOfSHAs`. Agree?

Reply:
Or are you talking about `linecount`? If so, it wasn't being reset but I agree that having one param pre-defined, and another one defined within `getShAArray` isn't good code. Changing that.

Reply:
> If so, it wasn't being reset but I agree that having one param pre-defined, and another one defined within getShAArray isn't good code. Changing that.

Thank you.

>If so, I agree we might need another name just in case something weird with the control flow happens. I mean, the problem you mentioned shouldn't happen because this is encapsulated inside an if, but I agree.

Ok great, defining it locally might be enough